src/struct_info_webgpu.json

```diff
@@ -0,0 +1,536 @@
+[
+    // ===========================================
+    // WebGPU
+    //   NOTE: This section is auto-generated.
```

Comment:
Maybe update the comment here?  "This file is ..."

Reply:
Done.